File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s)
{
    bool isBool = true;
    for(char &c : s)
    {
        if(!isdigit(c))
        {
            isBool = false;
            break;
        }
    }
    
    if(s.length() != 4 && s.length() != 6)
    {
        isBool = false;
    }
    
    return isBool;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cs
public class Solution 
{
    public bool solution(string s) 
    {
        if(s.Length != 4 && s.Length != 6)
        {
            return false;
        }
        
        foreach(char c in s)
        {
            if(c >= 65)
            {
                return false;
            }
        }
        
        return true;
    }
}